client/src/components/organisms/VideoStream/VideoStream.jsx

```diff
@@ -107,7 +123,7 @@ function VideoStream() {
                 </div>
```

Comment:
Every page that plays the video, has the creator's name. Link the creator's name to the Creator's page.

Reply:
Sure noted, will add in upcoming version :) Along with it, description section and notes are also gonna get included, so will add along with them.